packages/apify/src/actor.ts

```diff
@@ -1512,6 +1580,36 @@ export class Actor<Data extends Dictionary = Dictionary> {
         return Actor.getDefaultInstance().createProxyConfiguration(proxyConfigurationOptions);
     }
 
+    /**
+     * Charge for a specified number of events - sub-operations of the Actor.
+     *
+     * @param options The name of the event to charge for and the number of events to be charged.
+     */
+    static async charge(options: ChargeOptions): Promise<ChargeResult> {
+        return Actor.getDefaultInstance().charge(options);
+    }
+
+    /**
+     * Get the maximum amount of money that the Actor is allowed to charge.
+     */
+    static getMaxTotalChargeUsd(): number {
+        return Actor.getDefaultInstance().getMaxTotalChargeUsd();
+    }
+
+    /**
+     * Get the number of events with given name that the Actor has charged for so far.
+     */
+    static getChargedEventCount(eventName: string): number {
+        return Actor.getDefaultInstance().getChargedEventCount(eventName);
+    }
+
+    /**
+     * Get information about the pricing for this Actor.
+     */
+    static getPricingInfo(): ActorPricingInfo {
+        return Actor.getDefaultInstance().getPricingInfo();
+    }
```

Comment:
Do we really want those methods on the `Actor` class directly? Maybe better to expose the charging manager instead?

I remember the pushback when I added `getInputOrThrow` some time ago, I think this was supposed to be following the Actor specs? cc @netmilk @jancurn

Reply:
No idea, I have no problem adjusting this. AFAIK only `Actor.charge` is described in the whitepaper (https://whitepaper.actor/#charging-money)

Reply:
I feel the Actor namespace and the domain term are overloaded already, I'm in favor of organizing the non-essential (non-whitepaper, convenience) functionality in a nested/separate namespace.

@B4nan Would you mind sharing a snippet of how you would access the "charging manager", please?

Reply:
We would need a getter for it, now it's private, e.g. `Actor.getChargingManager().foo()`.